log-collector/src/main/scala/io/phdata/pulse/logcollector/LogCollectorRoutes.scala

```diff
@@ -106,7 +106,7 @@ class LogCollectorRoutes(solrStream: SolrCloudStream, kuduStream: Option[KuduMet
       entity(as[TimeseriesRequest]) { eventRequest =>
         kuduStream
           .map { client =>
-            eventRequest.payload.map(event => client.put(eventRequest.table_name, event))
+            eventRequest.payload.map(event => client.save(eventRequest.table_name, List(event)))
```

Comment:
Can you add a test that simulates a bad request to Kudu and verify a 500 error is returned?

Reply:
This test shoul go in `LogCollectorRoutesTest`